src/main/java/sirius/web/templates/NamedRowProcessor.java

```diff
@@ -28,5 +27,5 @@
      * @param lineNumber the line number which is currently being processed.
      * @param row        the data of the row / line to process
      */
-    void handleRow(int lineNumber, Map<String, Value> row);
+    void handleRow(int lineNumber, ListMultimap<String, Value> row);
```

Comment:
wouldn't it even be sexier to pass a "SmartRow" (or the like) interface here, which has getValue(String) -> Value and getValues(String) -> List<Value> to hide the implementation details.

This interface could be implemented in-place in SmartLineBaseProcessor